docs/src/appendix/scarb-toml.md

```diff
@@ -78,6 +78,17 @@ url = "http://your.second.rpc.url"
 block_id.number = "123"
 ```
 
+### `[tool.scarb]`
+
+```toml
+[tool.scarb]
+allow-prebuilt-plugins = ["snforge_std"]
+```
+Note: This configuration requires Scarb version >= 2.10.0 .
+
+It allows `scarb` to download precompiled dependencies used by `snforge_std` from [the registry](https://scarbs.xyz).
+The `snforge_std` library depends on a Cairo plugin that is written in Rust, and otherwise is compiled locally on the user's side.
```

Comment:
Let's keep it, but I think we need to add similar information to the installation docs. Right now we say we require Rust, but it should be explained that for new scarb versions it can be skipped for most platforms.